### PR TITLE
109 setup terraform pipelines to plan and deploy to ec2

### PR DESCRIPTION
**Pull request by KeManen**
Terraform is ready.

Waiting for next sprint to automate

### CODE REVIEW

**Comment by KeManen** [comment]
Add image creation and divide the module

**Comment by KeManen** [comment]
Waiting for changes #167 to work to get docker image ci for the pipeline as terraform is not for building dockers

**Comment by KeManen** [comment]
TODO:
- Check connection strings  between containers
- Figure out spotify dev settings
- Get Eddies secrets To the environments
- Do pipeline

**Comment by KeManen** [comment]
@EddieTheCubeHead Can you create an aws account for stagnum and add aws access key and secret to the github secrets in addition to the postgres secrets

**Comment by EddieTheCubeHead** [comment]
> @EddieTheCubeHead Can you create an aws account for stagnum and add aws access key and secret to the github secrets in addition to the postgres secrets

Looking into this

File: server/src/main.py [hunk 1]
```diff
@@ -23,13 +24,18 @@
 
 def _inject_secrets():
     secret_files = ["spotify_client_id", "spotify_client_secret"]
-    _logger.debug(f"Injecting secrets from the following files: {secret_files}")
+    _logger.debug(f"Injecting secrets from th_get_allowed_origins() e following files: {secret_files}")
```
[the remark this answers is not in the document]
Comment (EddieTheCubeHead) [reply server/src/main.py:27]: probably an erroneous tab-click adding a method call here

**Comment by KeManen** [comment]
NYT SE TAPAHTUU POJAT!

**Review by EddieTheCubeHead — APPROVED** [review]
LGTM, Elias fiksaa

**Pull request merged**